kml/serdes.cpp

```diff
@@ -269,7 +270,11 @@ void SaveStringsArray(KmlWriter::WriterWrapper & writer,
   for (auto const & s : stringsArray)
   {
     writer << offsetStr << kIndent2 << "<mwm:value>";
-    SaveStringWithCDATA(writer, s);
+    // Constants from our code do not need any additional checks or escaping.
+    if constexpr (std::is_same_v<StringViewLike, std::string_view>)
+      writer << s;
+    else
+      SaveStringWithCDATA(writer, s);
```

Comment:
Hm, what is the difference here? We can break some predefined behaviour depending on what is in SaveStringWithCDATA

Reply:
SaveStringWithCDATA may change user input that contains non-xml chars or < or & symbols. It is not the case for string_view: now it's only lang codes.

I've added ASSERT for that case.

Reply:
ASSERT_EQUAL then?

Reply:
Of course!